app/build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
     compile "com.android.support:cardview-v7:$rootProject.supportLibraryVersion"
     compile "com.google.android.gms:play-services-maps:$rootProject.playServicesVersion"
     compile "com.android.support:recyclerview-v7:$rootProject.supportLibraryVersion"
+    compile "com.android.support:gridlayout-v7:$rootProject.supportLibraryVersion"
```

Comment:
As the number buttons were static so I used Gridlayout for this task : https://github.com/dilpreet96/self-service-app/blob/bbc05af6a6814a4e568e59451e9b709981cae7f6/app/src/main/res/layout/activity_pass_code.xml#L55
We can use TableLayout also shall I update the PR with tablelayout?